creator-node/prod.env

```diff
@@ -23,7 +22,7 @@ logLevel=info
 maxAudioFileSizeBytes=1000000000
 pinAddCIDs=QmXSa6NxfA3e2hCpd8P5gNnZJs9PjZ3yGDHiRHR5B7Rq52,QmNU5Sv8se1aqfUFYFxupX1wdZfGTGnJp6vpoMbLNceGVi
 setTimeout=3600000
-snapbackHighestReconfigMode=RECONFIG_DISABLED
+stateMachineHighestReconfigMode=RECONFIG_DISABLED
```

Comment:
do we need this at all? like if the config inside the container has this we should just let that specify for 3p nodes

Reply:
fair - i guess this was jsut for explicitness, but i'm down to remove

Reply:
i think we should remove layers of indirection to make rollout simpler. the two issues i can potentially see here is if we forget to update this  only update config.js then it won't roll out correctly or if a node operator copies the env vars by hand every now and then (which some do) and they don't update this on time